pytheas-core/src/main/java/com/netflix/explorers/providers/FreemarkerTemplateProvider.java

```diff
@@ -68,13 +75,13 @@ public static void addUrlHandler(String protocol, URLStreamHandler handler) {
     
     private Configuration fmConfig = new Configuration();
     private boolean servletMode = false;
-    
-    @Inject(optional=true) ExplorerManager manager;
+    private ExplorerManager manager;
 
     @Context 
     private ThreadLocal<HttpServletRequest> requestInvoker;
-
-    public void setExplorerManager(ExplorerManager manager) {
+    
+    @Inject
+    public FreemarkerTemplateProvider(ExplorerManager manager) {
```

Comment:
Not having a default constructor is a breaking change and could affect existing code.  Can we keep a default constructor and add @Inject(optional=true) to a deprecated setExplorerManager?